site/pages/api/get-all-disruptions/__snapshots__/get-all-disruptions.test.ts.snap

```diff
@@ -9,6 +9,7 @@ exports[`getAllDisruptions > formatSortedDisruptions > correctly formats disrupt
   "displayId": "8fg3ha",
   "disruptionReason": "accident",
   "disruptionType": "planned",
+  "history": undefined,
```

Comment:
do we expect this to be undefined? we might need to update the mock data the test is using to include a mock history item.

Reply:
done